README.md

```diff
@@ -2328,6 +2328,21 @@ class API < Grape::API
 end
 ```
 
+For using Rack::Sendfile middleware, specify `to_path` method in your
```

Comment:
@dblock , thanks for comment! I mean that agrument of `file` dsl method should respond to `to_path` method (of couse `use Rack::Sendfile` shoule be applied in API class).
So it can be `FileSteamer` with realized `to_path` method or other object with `to_path` method. I uses "file streamer" notion for supporting article in the same style. 
I think that adding information about `Rack::Sendfile` will be overkill. I can replace information with this:

For using `Rack::Sendfile` middleware, specify `to_path` method in your file streamer class which returns path of served file:

```ruby
class FileStreamer
  # ...

  def to_path
    @file_path
  end

  # ...
end
```

Note: don't forget turn on `Rack::Sendfile` middleware in your API:

```ruby
class API < Grape::API
  use Rack::Sendfile
end
```

If you want to just add `Rack::Sendfile` and I starts to work see my comment on below :)